src/wallet/sqlite.cpp

```diff
@@ -400,11 +400,16 @@ SQLiteBatch::SQLiteBatch(SQLiteDatabase& database)
 void SQLiteBatch::Close()
 {
     // If m_db is in a transaction (i.e. not in autocommit mode), then abort the transaction in progress
+    bool force_conn_refresh = false;
     if (m_database.HasActiveTxn()) {
         if (TxnAbort()) {
             LogPrintf("SQLiteBatch: Batch closed unexpectedly without the transaction being explicitly committed or aborted\n");
         } else {
-            LogPrintf("SQLiteBatch: Batch closed and failed to abort transaction\n");
+            // In case we cannot abort the transaction, prevent other handler/s from dumping the ongoing
```

Comment:
In commit "sqlite: guard against dangling to-be-reverted db transactions" (508fda2fddfef0cefedde9777c914aec72adf7fb)

I found this comment confusing because I couldn't figure out when failing to abort a transaction would ever be expected to happen, and what the error handling strategy was supposed to be when it did happen. Would maybe suggest: "// If transaction cannot be aborted, it means there is a bug or there has been data corruption. Try to recover in this case by closing and reopening the database. Closing the database should also ensure that any changes made since the transaction was opened will be rolled back."

Reply:
Sure, done. Have slightly rephrased the suggestion. Thanks